includes/discovery/sensors/state/mrv-optidriver.inc.php

```diff
@@ -0,0 +1,408 @@
+/*
+ * Sensor State discovery module for the MRV® OptiDriver® Optical Transport Platform
+ *
+ * © 2017 Chris A. Evans <thecityofguanyu@outlook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+<?php
+
+// Optical Interface Power State
+if ($device['os'] == 'mrv-optidriver') {
+    $oids = snmpwalk_cache_multi_oid($device, 'nbsCmmcPortTable', array(), 'NBS-CMMC-MIB');
```

Comment:
Could I do `$oids = $pre_cache['mrv-od_oids']` here in lieu of the existing statement?